### PR TITLE
Update opengraph metadata for social cards

### PR DESCRIPTION
**Pull request by ghost**
I guess this should fix it

### CODE REVIEW

**Review by MylesBorins — APPROVED** [review]
LGTM

**Comment by MylesBorins** [comment]
@mlosh based on Contributing.md this should likely have waited a little bit longer for review from more collaborators before landing.

**Comment by ghost** [comment]
Well, this fixed facebook card, I think there is a conflict between opengraph and twitter tags. https://developer.twitter.com/en/docs/tweets/optimize-with-cards/guides/getting-started

**Pull request merged**
